### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by rjucf**

### CODE REVIEW

File: footer.php [hunk 1]
```diff
@@ -1,82 +1,30 @@
 		</main>
-		<footer class="primary-footer">
+		<footer class="bg-inverse pt-5 pb-5">
```
Comment (jmbarne3) [inline footer.php]: You can use `py-5` here instead of `pt-5` and `pb-5`.

File: header.php [hunk 2]
```diff
@@ -51,32 +51,38 @@
...
+
+					<div class="collapse navbar-collapse" id="navbarSupportedContent">
+						<?php
+						wp_nav_menu(
+							array(
+								'theme_location' => 'header-menu',
+								'container'      => 'false',
+								'menu_class'     => 'navbar-nav '.get_header_styles(),
+								'menu_id'        => 'header-menu',
+								'walker' => new Bootstrap_Walker_Nav_Menu(),
```
Comment (jmbarne3) [inline header.php]: There are some incompatibilities between the one we've used historically and BS4. So this navwalker might need to be adjusted slightly.

File: src/scss/_footer.scss [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    .address {
+      font-size: .8rem;
+    }
+  }
+
+  .primary-footer-section-center,
+  .primary-footer-section-right {
+    margin-top: 1.5rem;
+
+    @media(min-width: map-get($grid-breakpoints, md)) {
```
Comment (cjg89) [inline src/scss/_footer.scss]: You can use the `media-breakpoint-up(md)` mixin here from bootstrap if you want to:
https://v4-alpha.getbootstrap.com/layout/overview/#responsive-breakpoints

File: src/scss/_footer.scss [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+  .aascb-logo {
+    background-image: url('../img/aacsb.png');
+    height: 78px;
+    width: 78px;
+  }
+
+  //Social Icons
+  .social {
+
+    .social-menu-link {
```
Comment (cjg89) [inline src/scss/_footer.scss]: Hover states on these links would be nice to have

File: footer.php (hunk 1 above)
Comment (rjucf) [inline footer.php]: Thanks Jim: https://github.com/UCF/CBA-Theme/pull/3/commits/5acc5a221f92e9cd63e95fc9c356abe39e150233

**Comment by rjucf** [comment]
Thanks Jo: https://github.com/UCF/CBA-Theme/pull/3/commits/6297edd98961da8a5ed8666e5482bd35737b8f53

**Comment by rjucf** [comment]
Jim, you cool with my changes?

**Comment by jmbarne3** [comment]
Acceptable

**Pull request merged**